docs/hub-deployment-guide/new-cluster/external-account.md

```diff
@@ -0,0 +1,25 @@
+# AWS with external account
+
+In some cases, the organization has its own AWS account and is provided it to us to deploy the cluster.
+This method is far simpler as we don't have to handle cloud billing, since it is handled by the organization.
+
+There are some steps to do before the deploy:
+
+1. Instruct the external organization to create one AWS IAM account with full permissions.
+   Since we will have one account per engineer,
+   this should be for a specific engineer who is responsible for setting up the hub.
```

Comment:
```suggestion
   Since we will have one account per engineer,
   this initial account should be created for the specific engineer who is responsible for setting up the hub.
```